fxmanifest.lua

```diff
@@ -12,10 +13,6 @@ files {
     'html/style.css',
 }
 
-client_scripts {
-    'client.lua',
-}
+client_script 'client.lua'
 
-server_scripts {
-    'server.lua'
-}
+server_script 'server.lua'
```

Comment:
I don't think moving it all to one line makes it easier to read in my opinion since if a person new to FiveM scripting wants to add another file they may find it harder to add more LUA files and I am trying to make the HUD as easy as possible to use.

Reply:
Ill undo that change for the reason you stated, just did it so it doesn't look for other files when there isn't any spare ones